src/package/index.ts

```diff
@@ -0,0 +1,5 @@
+export * from './PackageInfo';
+export * from './PackageJSON';
+export * from './PackageStats';
+export * from './ResourceInfo';
+export * from '../virtual/VirtualPackage';
```

Comment:
No, that's not normal. I think this is a remnant of a refactoring. I originally had `VirtualPackage.ts` in the `src/package` folder -- so it was exporting it then. When I moved `VirtualPackage.ts` to the `src/virtual` folder, the refactoring feature must have rewritten that export and I did not notice it. We don't need it since we are exporting from `src/virtual`.